app/views/two_factor_authentications/_confirm.haml

```diff
@@ -0,0 +1,31 @@
+.col-md-12
+  .row
+    .col-md-12
+      %h3= t("two_factor_auth.confirm.title")
+      %p= t("two_factor_auth.confirm.status")
+      .small-horizontal-spacer
+      %h4= t("two_factor_auth.confirm.scan_title")
+  .row
+    .col-md-6
+      %p= t("two_factor_auth.confirm.scan_explanation")
+      = raw RQRCode::QRCode.new(@qrcode_uri).as_svg(padding: 0, module_size: 5)
```

Comment:
- Rails/OutputSafety: Tagging a string as html safe may be a security risk.
- Avoid using instance variables in partials views